packages/swapper/src/swappers/PortalsSwapper/utils/fetchPortalsTradeOrder.ts

```diff
@@ -108,8 +127,13 @@ export const fetchPortalsTradeOrder = async ({
     return response.data
   } catch (error) {
     if (axios.isAxiosError(error)) {
+      if ((error as AxiosError<{ message: string }>).response?.data?.message) {
+        throw new PortalsError((error as AxiosError<{ message: string }>).response?.data.message!)
+      }
```

Comment:
[nitpick]: Non-null assertion avoidable here, and be dry-er

```suggestion
      const message = (error as AxiosError<{ message: string }>).response?.data?.message
      if (message) {
        throw new PortalsError(message)
      }
```

Reply:
🧠  b79caa23e9